features/netsocket/emac-drivers/TARGET_NUVOTON_EMAC/TARGET_M480/m480_eth.c

```diff
@@ -262,6 +262,8 @@ void numaker_eth_init(uint8_t *mac_addr)
                     EMAC_CAMCTL_ABP_Msk;
     EMAC->CAMEN = 1;    // Enable CAM entry 0    
 
+    /* Limit the max receive frame length to 1514 + 4 */
+    EMAC->MRFL = 1518;    
```

Comment:
Should this not be expressed based on `NU_ETH_MAX_FLEN` or `PACKET_BUFFER_SIZE`?

It might be that those are not quite defined right - I would expect the `NU_ETH_MAX_FLEN` to be 1518, not 1520, and then maybe PACKET_BUFFER_SIZE should be that rounded up to 1520 (multiple of 4).

Reply:
@kjbracey-arm,  thanks of your advisement, I correct the definition & relative code in this commit 42103f0.